litellm/llms/prompt_templates/factory.py

```diff
@@ -1052,7 +1076,7 @@ def prompt_factory(
             messages=messages, prompt_format=prompt_format, chat_template=chat_template
         )
     elif custom_llm_provider == "gemini":
-        if model == "gemini-pro-vision":
+        if "vision" in model or "gemini-1.5-pro" in model:
```

Comment:
@cheahjs  does gemini-1.5-pro (not vision) support images?

Reply:
Hey @cheahjs can you add a test to make sure this works out here - https://github.com/BerriAI/litellm/blob/71db88115d96348c6273a7957639b4c67c9761f9/litellm/tests/test_completion.py#L2356